modules/rubiconAnalyticsAdapter.js

```diff
@@ -62,15 +62,20 @@ const cache = {
 
 const BID_REJECTED_IPF = 'rejected-ipf';
 
-export let rubiConf = {
-  pvid: generateUUID().slice(0, 8),
-  analyticsEventDelay: 0,
-  dmBilling: {
-    enabled: false,
-    vendors: [],
-    waitForAuction: true
+export let rubiConf;
+export const resetRubiConf = () => {
+  rubiConf = {
+    pvid: generateUUID().slice(0, 8),
+    analyticsEventDelay: 0,
+    dmBilling: {
+      enabled: false,
+      vendors: [],
+      waitForAuction: true
+    }
   }
-};
+}
+resetRubiConf();
```

Comment:
can yoy explain why there was the need for this addition?

Reply:
it is for tests to be able to start clean, and I figured instead of defaulting and then having a function reset those defaults, better to just have a function to set defaults and call it at load time.

But yeah, tests use it to clean the conf between tests.